crystal-map-processor/src/main/java/com/schwarz/crystalprocessor/generation/CodeGenerator.kt

```diff
@@ -30,6 +37,59 @@ class CodeGenerator(private val filer: Filer) {
         }
     }
 
+    @Throws(IOException::class)
+    fun generateAndFixAccessors(
+        entityToGenerate: FileSpec,
+        generateAccessors: MutableList<CblGenerateAccessorHolder>,
+        processingEnvironment: ProcessingEnvironment
+    ) {
+        ClassName(entityToGenerate.packageName, entityToGenerate.name)?.apply {
+            ProcessingContext.createdQualifiedClazzNames.add(this)
+        }
+
+        val codePath = processingEnvironment.options[CoachBaseBinderProcessor.KAPT_KOTLIN_GENERATED_OPTION_NAME]
+        val fileWithHeader = entityToGenerate.toBuilder().addFileComment(HEADER).build()
+
+        val fixedFileString = generateAccessors.fold(fileWithHeader.toString()) { acc, generateAccessor ->
+            if (generateAccessor.memberFunction != null && generateAccessor.memberFunction.isSuspend) {
+                acc.replace(Regex("(${generateAccessor.memberFunction.name}\\([^)]*\\)):\\s*Unit(\\s*=)"), "$1$2")
+            } else {
```

Comment:
Can we extract the Regex to a const with a describing name?

Reply:
Unfortunately no, since the memberFunction name has to be added dynamically